src/Models/Tag.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Astrogoat\Blog\Models;
+
+use Helix\Fabrick\Icon;
+use Helix\Lego\Models\Contracts\Sectionable;
+use Helix\Lego\Models\Model as LegoModel;
+use Helix\Lego\Models\Traits\HasSections;
+
+class Tag extends LegoModel implements Sectionable
+{
+    use HasSections;
+
+    protected $table = 'article_tags';
+
+    public static function icon(): string
+    {
+        return Icon::DOCUMENT_TEXT;
+    }
+
+    public function editorShowViewRoute(string $layout = null): string
+    {
+//        return route('lego.blog.tag.editor', [
+//            'tag' => $this,
```

Comment:
I'd seen the discussion between you and BaffourAdu on this. I'd recommend removing the current comment and possibly put a 'unused' type comment just for clarity.